tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -800,8 +800,9 @@ public async Task MultiTenantWithAuthorityOverrideAsync()
 
                 // Act
                 var accounts = await pca.GetAccountsAsync().ConfigureAwait(false);
+                var account = accounts.Single(a => a.HomeAccountId.TenantId == tenant1);
                 AuthenticationResult response = await
-                    pca.AcquireTokenSilent(new[] { "User.Read" }, accounts.First())

```

Comment:
`.First()` is not deterministic because the underlying collection is a dictionary.